From 4caacf4577e615a80c4282c700347dfb2e931788 Mon Sep 17 00:00:00 2001
From: Ben Boeckel <ben.boeckel@kitware.com>
Date: Mon, 15 Feb 2021 10:17:53 -0500
Subject: [PATCH 2/2] Adaptors/Cam: include the .txx file

This allows the TU to instantiate the template information as needed.
---
 Adaptors/Cam/fv_create_data.cxx | 1 +
 Adaptors/Cam/se_create_data.cxx | 1 +
 2 files changed, 2 insertions(+)

diff --git a/Adaptors/Cam/fv_create_data.cxx b/Adaptors/Cam/fv_create_data.cxx
index 95dc7260f0..0dc45c3e33 100644
--- a/Adaptors/Cam/fv_create_data.cxx
+++ b/Adaptors/Cam/fv_create_data.cxx
@@ -16,6 +16,7 @@
 #include "fv_create_data.h"
 
 #include "Grid.h"
+#include "Grid.txx"
 
 #include "vtkCPDataDescription.h"
 #include "vtkCPInputDataDescription.h"
diff --git a/Adaptors/Cam/se_create_data.cxx b/Adaptors/Cam/se_create_data.cxx
index 4552340652..a9877410df 100644
--- a/Adaptors/Cam/se_create_data.cxx
+++ b/Adaptors/Cam/se_create_data.cxx
@@ -16,6 +16,7 @@
 #include "se_create_data.h"
 
 #include "Grid.h"
+#include "Grid.txx"
 
 #include "vtkCPDataDescription.h"
 #include "vtkCPInputDataDescription.h"
-- 
GitLab
